docs/learn-treasury.md

```diff
@@ -2,24 +2,67 @@
- - Ecosystem provisions (collaborations with friendly chains).
- - Marketing activities (advertising, paid features, collaborations).
- - Community events and outreach (meetups, pizza parties, hackerspaces).
- - Software development (wallets and wallet integration, clients and client upgrades).
+- Infrastructure deployment and continued operation.
+- Network security operations (monitoring services, continuous auditing).
+- Ecosystem provisions (collaborations with friendly chains).
+- Marketing activities (advertising, paid features, collaborations).
+- Community events and outreach (meetups, pizza parties, hackerspaces).
+- Software development (wallets and wallet integration, clients and client upgrades).
 
-The treasury is ultimately controlled by the stakeholders, and how the funds will be spent is up to their judgment.
+The treasury is ultimately controlled by the council, and how the funds will be spent is up to their judgment.
 
-## Resources
+## Funding the Treasury
+
+The Treasury is funded from different sources:
+
+1. Slashing: When a validator is slashed for any reason, the slashed amount is sent to the Treasury with a reward going to the entity that reported the validator (another validator or fisherman). The reward is taken from the slash amount and varies per offence and number of reporters.
+2. Transaction fees: A portion of each block's transaction fees goes to the Treasury, with the remainder going to the block author.
+3. Staking inefficiency: [Inflation](learn-staking#inflation) is designed to be 10% in the first year, and the ideal staking ratio is set at 50%, meaning half of all tokens should be locked in staking. Any deviation from this ratio will cause a proportional amount of the inflation to go to the Treasury. In other words, if 50% of all tokens are staked, then 100% of the inflation goes to the validators as reward. If the staking rate is greater than or less than 50%, then the validators will receive less, with the remainder going to the Treasury.
+4. Parathreads: [Parathreads](learn-parathreads) participate in a per-block auction for block inclusion. Part of this bid goes to the validator that accepts the block and the remainder goes to the Treasury.
+
+## Creating a Treasury Proposal
+
+The proposer has to deposit 5% of the requested amount or 20 KSM (whichever is higher) as an anti-spam measure. This amount is burned if the proposal is rejected, or refunded otherwise. These values are subject to [governance](learn-governance) so they may change in the future.
+
+### Announcing the Proposal
+
+To minimize storage on chain, proposals don't contain contextual information. When a user submits a proposal, they will probably need to find an off-chain way to explain the proposal. Most discussion takes place on the following platforms:
+
+- [Commonwealth.im](https://commonwealth.im) which allows users to log in with their KSM address and automatically reads proposals from the chain, turning them into discussion threads.
```

Comment:
Let's move this to the bottom of the list (guessing Riot is actually the most used?)
```suggestion
- [Commonwealth.im](https://commonwealth.im) is a community site that allows users to log in with their KSM address and automatically reads proposals from the chain, turning them into discussion threads.
```